x-pack/plugin/logsdb/build.gradle

```diff
@@ -42,3 +42,9 @@ tasks.named("javaRestTest").configure {
 tasks.named('yamlRestTest') {
   usesDefaultDistribution()
 }
+
+tasks.named("yamlRestCompatTestTransform").configure ({ task ->
```

Comment:
I had to skip these yaml tests here because they expect synthetic source recovery to be used.